athenahealth/patients.go

```diff
@@ -307,6 +309,7 @@ type UpdatePatientOptions struct {
 	FirstName           *string
 	GenderIdentity      *string
 	GenderIdentityOther *string
+	HasMobile           *bool
```

Comment:
Any reason the client field name doesn't match the API field name? Seems like most others are 1-to-1

Reply:
copied it from the patient record
suppose there's no reason on to just match the API fields directly